sympy/matrices/expressions/tests/test_matmul.py

```diff
@@ -184,3 +184,8 @@ def test_shape_error():
     A = MatrixSymbol('A', 2, 2)
     B = MatrixSymbol('B', 3, 3)
     raises(ShapeError, lambda: MatMul(A, B))
+
+def test_issue_9503():
```

Comment:
you could also put the link of the issue

Reply:
I have added the issue link, thank you for reviewing!